tests/features/tmgmt/tmgmt_workbench.feature

```diff
@@ -26,7 +26,7 @@ Feature: TMGMT Workbench features
     And I should see the text "The current piece of content's moderation state is: published"
     And I should see "Not translated" in the "French" row
     And I should see "Not translated" in the "Italian" row
-    And I select the radio button "" with the id "edit-languages-it"
```

Comment:
Sadly, these generated radio buttons don't have a label. That results now, always, in an exception. I've created a new step that doesn't require a label.
A better fix would be to add a label. That should be done in DO and it's out of scope.

Reply:
👍 
Your implementation is good and will not become useless if the label use is fixed in DO. 

Indeed, the label tag added to a radio button is strongly recommended in HTML but not mandatory. 

So, your step is and will be useful for case where the label tag is not present and we cannot fix it because belongs to a 3rd party code.

That does not mean we should not raise the point of the label tag to the DO community but as you said it very well, it is out of the scope of the current PR. 😄